frontend/src/atoms.tsx

```diff
@@ -0,0 +1,3 @@
+import { atom } from 'jotai'
+
+export const storeAtom = atom<boolean>(false)
```

Comment:
```suggestion
export const storeVisibleAtom = atom(false)
```
처럼 쓰면 좀 더 명확할 것 같습니다.